test/foundry/TrancheWrapper.t.sol

```diff
@@ -0,0 +1,321 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.10;
+
+import "forge-std/Test.sol";
+
+import "../../contracts/interfaces/IERC20Detailed.sol";
+import "../../contracts/TrancheWrapper.sol";
+
+contract TestTrancheWrapper is Test {
+    using stdStorage for StdStorage;
+
+    uint256 internal constant MAINNET_CHIANID = 1;
+    uint256 internal constant ONE_TRANCHE_TOKEN = 1e18;
+
+    address internal constant IDLE_CDO_ADDRESS = 0x46c1f702A6aAD1Fd810216A5fF15aaB1C62ca826;
+    address internal constant IDLE_TRANCHE_ADDRESS = 0x852c4d2823E98930388b5cE1ed106310b942bD5a;
+
+    address public owner;
+    IdleCDO internal idleCDO;
+    IERC20Detailed internal underlying;
+    IERC20Detailed internal strategyToken;
+    IdleCDOTranche internal AAtranche;
+    IdleCDOTranche internal BBtranche;
+    IIdleCDOStrategy internal strategy;
+
+    uint256 internal decimals;
+    uint256 internal ONE_SCALE;
+    uint256 internal initialBal;
+    bytes internal extraData;
+    bytes internal extraDataSell;
+
+    IERC20Detailed internal tranche;
+    TrancheWrapper internal trancheWrapper;
+
+    modifier runOnForkingNetwork(uint256 networkId) {
```

Comment:
I've found out that we can simply do something like this https://github.com/Idle-Labs/idle-tranches/blob/fec645f837c4d52c8697dd15fe047ec27abbd50a/test/foundry/IdleTokenFungible.t.sol#L48-L50 and avoid the modifier altogether and then tests can simply be launched with `forge test -vvv` without block number and rpc endpoint on cli params. I would use this pattern from now on if possibile